src/checkbox/checkbox.component.spec.ts

```diff
@@ -25,13 +27,32 @@ class CheckboxTest {
 	onIndeterminateChange() {}
 }
 
-describe("Checkbox", async() => {
+const defaults = {
```

Comment:
Typically we try to [provide defaults within the tests](https://github.ibm.com/Watson-Health/design-library/blob/carbon-x/packages/pattern-library/src/components/modal/tests/requirements_exp.js#L42) themselves so that developers using them only have to overwrite them if needed. In which most cases, developers will only have to overwrite the root as the rest of the elements are always searched for from the root element. (e.g: https://github.ibm.com/Watson-Health/design-library/blob/carbon-x/packages/pattern-library/src/components/modal/tests/modal.spec.js#L66-L68)

Reply:
That makes sense 👍